lua/telescope/health.lua

```diff
@@ -45,10 +45,12 @@ local required_plugins = {
 local check_binary_installed = function(package)
   local binaries = package.binaries or { package.name }
   for _, binary in ipairs(binaries) do
-    if is_win then
+    local found = vim.fn.executable(binary) == 1
+    if not found and is_win then
       binary = binary .. ".exe"
+      found = vim.fn.executable(binary) == 1
```

Comment:
```suggestion
      found = vim.fn.executable(binary .. ".exe") == 1
```
maybe instead of overriding `binary`, we can do this. Then the `io.popen()` will be without `.exe` and hopefully it will actually work and return the version number?

Reply:
In my environment, this code don't change the version result.
This line will be executed if `rg` does not exist.
Therefore,  when `rg` does not exist and `rg.exe` exists , does executing `rg --version` output unexpected error?

Reply:
ok, let's just keep things as is then. thanks for trying.